pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/LLCRealtimeClusterIntegrationTest.java

```diff
@@ -19,58 +19,56 @@
 package org.apache.pinot.integration.tests;
 
 import com.fasterxml.jackson.databind.JsonNode;
-import com.google.common.base.Function;
 import java.io.File;
 import java.io.IOException;
 import java.util.Arrays;
 import java.util.Collections;
-import java.util.HashMap;
 import java.util.List;
-import java.util.Map;
 import java.util.Random;
 import org.apache.avro.reflect.Nullable;
 import org.apache.commons.configuration.Configuration;
 import org.apache.commons.io.FileUtils;
 import org.apache.helix.ZNRecord;
-import org.apache.pinot.common.config.IndexingConfig;
-import org.apache.pinot.common.config.SegmentsValidationAndRetentionConfig;
 import org.apache.pinot.common.config.TableConfig;
-import org.apache.pinot.common.config.TableCustomConfig;
 import org.apache.pinot.common.config.TableNameBuilder;
-import org.apache.pinot.common.config.TenantConfig;
 import org.apache.pinot.common.utils.CommonConstants;
 import org.apache.pinot.common.utils.CommonConstants.Helix.TableType;
 import org.apache.pinot.controller.ControllerConf;
 import org.apache.pinot.util.TestUtils;
-import org.testng.Assert;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
 
 /**
  * Integration test that extends RealtimeClusterIntegrationTest but uses low-level Kafka consumer.
  */
 public class LLCRealtimeClusterIntegrationTest extends RealtimeClusterIntegrationTest {
-
-  public static final String CONSUMER_DIRECTORY = "/tmp/consumer-test";
-  public static final long RANDOM_SEED = System.currentTimeMillis();
-  public static final Random RANDOM = new Random(RANDOM_SEED);
-
-  public final boolean _isDirectAlloc = RANDOM.nextBoolean();
-  public final boolean _isConsumerDirConfigured = RANDOM.nextBoolean();
-  private final long _startTime = System.currentTimeMillis();
-
+  private static final String CONSUMER_DIRECTORY = "/tmp/consumer-test";
```

Comment:
Do we need to override the getServer method if split commit is enabled? Otherwise the server will not start with the config

Reply:
We don't because getServer() will call overrideServerConf() where we override the config